recommender/static/html/recommender.html

```diff
@@ -76,18 +78,15 @@
 <div class="recommenderBlock">
     <div class="recommenderRecommendations">
         <div class="recommenderContent">
-            <div class="recommender_hideShow" role='button' tabindex='0' data-step="1" aria-expanded='false' data-intro="This is a list of recommended resources. If you're stuck, curious, or want to learn more, please look around. If you find a useful resources, help your fellow classmates by adding it!">
+            <div class="recommender_hideShow" role='button' tabindex='0' data-step="1" aria-expanded='false' data-intro="${_("This is a list of recommended resources. If you're stuck, curious, or want to learn more, please look around. If you find a useful resources, help your fellow classmates by adding it!")}">
                 Show related resources
```

Comment:
@OmarIthawi is there a reason we're not translating some texts like this one? or is it just missed?

Reply:
I missed it.

Reply:
I've added them all @OmarIthawi 👍🏼

Reply:
Thanks for the help :)